src/rdkafka.c

```diff
@@ -3565,12 +3569,12 @@ int rd_kafka_queue_poll_callback (rd_kafka_queue_t *rkqu, int timeout_ms) {
 static void rd_kafka_toppar_dump (FILE *fp, const char *indent,
 				  rd_kafka_toppar_t *rktp) {
 
-	fprintf(fp, "%s%.*s [%"PRId32"] leader %s\n",
+	fprintf(fp, "%s%.*s [%"PRId32"] broker %s\n",
```

Comment:
it would be useful to also print the leader id here

Reply:
i added leader_id. Also added broker_id because timing of that is consistent with leader_id and broker_id may not align with (broker *). please confirm this makes sense to you.

Reply:
consistent with the statistics, on second thoughts I've removed broker_id.